Consolidation OOP/main.py

```diff
@@ -0,0 +1,22 @@
+from consolidate import *
+import glob
+import matplotlib.pyplot as plt
+import pdb
+from PIL import Image
```

Comment:
The only import that is necessary here seems to be consolidate, please remove the others.